kivy/cache.py

```diff
@@ -163,14 +163,18 @@ def remove(category, key=None):
         try:
             if key is not None:
                 del Cache._objects[category][key]
+                Logger.trace('Cache: Removed %s:%s from cache' %
+                             (category, key))
             else:
                 Cache._objects[category] = {}
+                Logger.trace('Cache: Flushed category %s from cache' %
+                             category)
         except Exception:
             pass
 
     @staticmethod
     def _purge_oldest(category, maxpurge=1):
-        print('PURGE', category)
+        Logger.debug('Cache: Remove oldest in %s' % category)
```

Comment:
Shouldn't all of them be trace rather than debug?

Reply:
I think that it makes sense from a developer perspective to use debug for this function and the ```_purge_by_timeout``` function, as it is useful to know when those two function are called when debugging the cache functionality without flooding the terminal with too much logging data.

However if the maintainers disagree, then I'll be happy to change it, as long the cache debugging is actually logged and not just printed to stdout.